www/js/reducers/points.js

```diff
@@ -0,0 +1,16 @@
+
+import {ADD_POINT, RESET_POINTS} from '../actions/point-actions'
+
+export function points(state = [], action) {
+  switch(action.type) {
+    case ADD_POINT:
+      return [
+        ...state,
+        Object.assign({}, action.point)
```

Comment:
`Object.assign()` makes copies of objects. However we actually don't need it here. I think we only need to make copies of objects originating from the state argument.